src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -60,9 +60,11 @@ internal class ConnectionManager : IConnectionManager {
                 .Add(_statusBarViewModel)
                 .Add(_hostLoadIndicatorViewModel)
                 .Add(() => _sessionProvider.BrokerStateChanged -= BrokerStateChanged)
+                .Add(() => _sessionProvider.SessionStateChanged -= SessionStateChanged)
```

Comment:
I don't think we want to track any session (i.e. intellisense and PM). We are interested in REPL session only.